experiments/conductor/cmd/runner/controller_commands.go

```diff
@@ -339,8 +339,8 @@ Respond only with the YAML content, no explanations.`,
 
 	cfg := CommandConfig{
 		Name:         "Generate Create YAML",
-		Cmd:          "codebot",
-		Args:         []string{"--prompt=/dev/stdin"},
+		Cmd:          "gemini",
+		Args:         []string{"--yolo", "--prompt=/dev/stdin"},
```

Comment:
Can you touch base with @barney-s? Wondering if we can do this using the sandbox option he looked into?

Reply:
Updated the code to use sandbox by default.

Will make the sandbox image configurable in the follow up PR.